maps/torch/torch3_deck3.dmm

```diff
@@ -12563,9 +12576,14 @@
 /obj/effect/floor_decal/corner/paleblue{
 	dir = 10
 	},
-/obj/machinery/atmospherics/pipe/simple/hidden/supply{
-	dir = 9;
-	pixel_y = 0
+/obj/structure/cable/green{
+	d1 = 2;
+	d2 = 4;
+	icon_state = "2-4"
+	},
+/obj/machinery/atmospherics/pipe/simple/hidden/scrubbers,
+/obj/machinery/atmospherics/pipe/manifold/hidden/supply{
+	dir = 4
 	},
```

Comment:
This is causing unit tests to fail, since it's leaking.

`## UNIT_TEST ##: [MAP: Station pipes shall not leak]: Following pipe is leaking: Air supply pipe manifold (/obj/machinery/atmospherics/pipe/manifold/hidden/supply) ([0x2003bc3]) (the deck) (53,98,3) (/turf/simulated/floor/tiled)`